src/pages/docs/infrastructure/workers/index.mdx

```diff
@@ -89,6 +90,7 @@ Learn more about configuring and using [dynamic worker pools](/docs/infrastructu
 An **External Worker** is either:
 - A [Windows](/docs/infrastructure/deployment-targets/tentacle/windows/) or [Linux](/docs/infrastructure/deployment-targets/tentacle/linux) Tentacle.
 - An [SSH machine](/docs/infrastructure/deployment-targets/linux/ssh-target) that has been registered with the Octopus Server as a worker.  
+- A [Kubernetes Worker](/docs/infrastructure/workers/autoscaling-workers) that has been installed in a k8s cluster, and has self-registered with the Octopus Server
```

Comment:
A link to here would make more sense to me: `https://octopus.com/docs/infrastructure/workers#installing-a-kubernetes-worker`

It was very confusing getting taken to a page about autoscaling from here.

If I'm thinking about this from a user who knows they want workers in k8s, but doesn't not what that entails - they'll want to click through this page and the installation process makes the most sense here

Reply:
I don't think we use `k8s` elsewhere either?

Reply:
Removed k8s.
The other links point to deployment target pages which explain the type of tentacle.
Given our "Auto Scaling worker" has been renamed  - is it ok if this link points there?

Reply:
Ah, I see, what a headache.

Can there be a backlink on the `Auto Scaling worker` page for installation? :joy: